plugins/modules/junos_vlan.py

```diff
@@ -172,9 +225,9 @@ def main():
     """
     element_spec = dict(
         name=dict(),
-        vlan_id=dict(type="int"),
+        vlan_id=dict(type="int", required=True),
```

Comment:
Top level keys cannot be required for modules having `aggregate` option

Reply:
Done